src/js/components/homepage/visualizations/treemap/TreeMap.jsx

```diff
@@ -6,15 +6,19 @@
 import React from 'react';
 import * as d3 from 'd3';
 import _ from 'lodash';
+import * as MoneyFormatter from 'helpers/moneyFormatter';
+import * as Icons from 'components/sharedComponents/icons/Icons';
 
 import TreeMapCell from './TreeMapCell';
-import TreeMapSidebar from './TreeMapSidebar';
-import * as Icons from '../../../sharedComponents/icons/Icons';
+import TreeMapTooltip from './TreeMapTooltip';
+import SubTreeMap from './SubTreeMap';
 
 const propTypes = {
     categories: React.PropTypes.object,
     descriptions: React.PropTypes.array,
-    colors: React.PropTypes.array
+    colors: React.PropTypes.array,
+    subfunctions: React.PropTypes.object,
+    alternateColors: React.PropTypes.array
 };
 
 export default class TreeMap extends React.Component {
```

Comment:
Agree, it's become a little scary.  Appreciate the suggestion on where to start separating it!